### ISSUE TITLE
Spec document has wrong header level for classes and properties

### ISSUE THREAD

**Issue by RubenVerborgh** [HydraCG/Specifications#78]
The generated lists of [classes](http://www.hydra-cg.com/spec/latest/core/#classes) and [properties](http://www.hydra-cg.com/spec/latest/core/#properties) start out with two `<h2>` elements, but the remaining elements are `<h3>`. This results in an incorrect structure:

![image](https://cloud.githubusercontent.com/assets/675313/5529240/08ce4476-8a12-11e4-8dee-4f8c8fe7806e.png)

The respec-w3c-extensions.js script correctly generates `<h3>` elements, but ReSpec changes (only) the first two of them to `<h2>`. Since I'm not familiar enough with ReSpec to know which one is to blame, I decided to post it here. Tried looking for a fix, but didn't find a satisfactory one right away (subsection numbers started to appear).

**Comment by alien-mcl** [comment]
I think this can be closed without any actions. It seems the spec displays heads properly (`Classes` is a bit larger than actual class names)
![specscreenshot](https://user-images.githubusercontent.com/8111925/51077589-0fb03100-16a9-11e9-9dc5-060836590788.png)

**Comment by tpluscode** [comment]
Indeed, I don't see a problem with current doc. The header is `<h2>` and its sub-points are `<h3>`.

My vote to close

**Comment by akuckartz** [comment]
According to https://github.com/w3c/respec/issues/386#issuecomment-71359609 it was not a ReSpec issue.

**Comment by alien-mcl** [comment]
This would mean that the issue is random I was neither able to reproduce in any attempt, nor to find that getJSON async call that was mentioned in the respec issue.  This issue is 3 years old and I'm keen to close it. Any objections?
